#! /bin/sh /usr/share/dpatch/dpatch-run
## manual-section-for-drv.dpatch by <q-funk@ubuntu.com>
##
## DP: fix incorrect man section 7 -> 1 as reported by Lintian.
@DPATCH@
--- a/man/Makefile	2009-05-13 21:39:17.000000000 +0000
+++ b/man/Makefile	2009-09-08 21:42:14.000000000 +0000
@@ -24,6 +24,7 @@ MAN1	=	cancel.$(MAN1EXT) \
 		cups-config.$(MAN1EXT) \
 		cupstestdsc.$(MAN1EXT) \
 		cupstestppd.$(MAN1EXT) \
+		drv.$(MAN1EXT) \
 		lp.$(MAN1EXT) \
 		lpoptions.$(MAN1EXT) \
 		lppasswd.$(MAN1EXT) \
@@ -47,7 +48,6 @@ MAN5	=	classes.conf.$(MAN5EXT) \
 		printers.conf.$(MAN5EXT) \
 		subscriptions.conf.$(MAN5EXT)
 MAN7	=	backend.$(MAN7EXT) \
-		drv.$(MAN7EXT) \
 		filter.$(MAN7EXT) \
 		notifier.$(MAN7EXT)
 MAN8	=	cupsaccept.$(MAN8EXT) \
debian/patches/debian/patches/manual-section-for-drv.dpatch
